client/hotrod-client/src/test/java/org/infinispan/client/hotrod/retry/ClientListenerFailoverBusyTest.java

```diff
@@ -158,7 +153,6 @@ public Object visitPutKeyValueCommand(InvocationContext ctx, PutKeyValueCommand
          executor.submit(() -> {
             try {
                barrier.await();
```

Comment:
are we sure that any interceptor is executed by a different thread?
Maybe the CountDownLatch would be better.